### PR TITLE
intro: split intro slides up into multiple files

### PR DESCRIPTION
**Pull request by chrbeckm**
implement first changes from Klausurtagung

- s. 7 Roadmap @aknierim 
- s. 8 adapt plan to proposed changes in order
- s. 16 list advantages of using Linux, explanation why it looks different on every of our laptops 
- s. 20 + 21: If the data literacy class takes place remove, otherwise need to shorten it
- s. 22 add screenshots
- s. 23 Vim or nVim?
- s.25 dir structure tikz picture from @aknierim 
- short list of `mkdir`, `touch`, `ls` (cheat sheet spoiler)

### CODE REVIEW

**Comment by aknierim** [comment]
Roadmap added in https://github.com/pep-dortmund/toolbox-workshop/pull/396/commits/af421f860b7165e14b0e23eb848d4cf14f1765bf, https://github.com/pep-dortmund/toolbox-workshop/pull/396/commits/d2f62ff0cd935cd1de5f3e9d17a46e450262d39c, and https://github.com/pep-dortmund/toolbox-workshop/pull/396/commits/dd10b628fd86d753342e9b1ca971a2b3823f1539.

**Comment by aknierim** [comment]
>     * s. 23 Vim or nVim?

Personally speaking, I would be in favor of Neovim, especially since plugin managers such as [lazy.nvim](https://github.com/folke/lazy.nvim) make it very easy to add plugins and customize the editor to your liking/needs.

**Comment by SepplL** [comment]
> 
> Personally speaking, I would be in favor of Neovim, especially since plugin managers such as [lazy.nvim](https://github.com/folke/lazy.nvim) make it very easy to add plugins and customize the editor to your liking/needs.

On the other hand `Vim` has the bigger history and our claim of being available on most systems holds more for `vim` than `nvim`

But I can totally see the reasoning of "updating" the slides to keep them more towards what more and more people use nowadays.

**Comment by SepplL** [comment]
Also I added some first ideas for slides 16 and 22 (now 23 with added slide for previous s.16)
Def. open to discussion and improvements as
- use another script
- make better/bigger images
- more reasons on why linux
- changing from `vim` to `nvim`

**Comment by chrbeckm** [comment]
> On the other hand `Vim` has the bigger history and our claim of being available on most systems holds more for `vim` than `nvim`
> 
> But I can totally see the reasoning of "updating" the slides to keep them more towards what more and more people use nowadays.

I would argue for that too, with regard to compatibility we would need to think about `vi` as well

**Comment by SepplL** [comment]
Do we still want to add a Code/VSCode Screenshot for comparison?

**Comment by SepplL** [comment]
> 
> I would argue for that too, with regard to compatibility we would need to think about `vi` as well

`vim` is available pretty much everywhere, where `vi` exists too.
If you want that type of backwards-compatibility, we might as well make a dive into `sed` and all `lisp`-based editors as well.
I don't think it is necessary to go back in time that far. Tbh, all modern systems ship `Neovim` without any issue as well. And given the `appimage` available in the `Neovim` repo, it is very straight forward to get it working on most remote systems as well.

Tl;dr: I'm in favor of mentioning `vim` as the starting point (albeit historically not accurate), but `vi` is maybe a bit too much.

**Comment by chrbeckm** [comment]
> Do we still want to add a Code/VSCode Screenshot for comparison?
> 

I would say no, and maybe have a nvim screenshot with one of the python explanation boxes in it, to show even more features.

**Comment by aknierim** [comment]
Directory structure tikz graphics are now reworked

**Comment by SepplL** [comment]
Looks good!
Any preferences on showing the structure in a tree command at a later stage?

**Comment by SepplL** [comment]
Do we want to leave out Neovim entirely for the screenshot comparison?

**Comment by chrbeckm** [comment]
I wouldn't overload it, and two screenshots should be enough. Additionally, we let them install VSCode/Codium and I thought it would be good to show that it has all the features.
We could do a second slide with more screenshots, or use nVim in the demonstration on how to exit and show it there.

**Comment by SepplL** [comment]
Good point.
Then I would move it to the demonstration to not overload the intro as a whole.
The benefit of showing it live is greater than the slide, I guess.

**Comment by aknierim** [comment]
> The benefit of showing it live is greater than the slide, I guess.

I second that.

**Comment by SepplL** [comment]
> I wouldn't overload it, and two screenshots should be enough. Additionally, we let them install VSCode/Codium and I thought it would be good to show that it has all the features. 

Also given the main point in the slide is to show, what a bad editor can look like, presenting their install is more useful.

**Comment by chrbeckm** [comment]
> So, all in all, we are close to being done with the intro?
> What is missing currently?

Definitely the answers from this years questionnaire. I can't think of other points right now.

**Review by ChAr-De — APPROVED** [review]
